src/enforcer/HolographRoyalties.sol

```diff
@@ -297,18 +297,13 @@ contract HolographRoyalties is Admin, Owner, Initializable {
     address payable[] memory addresses = _getPayoutAddresses();
     uint256[] memory bps = _getPayoutBps();
     uint256 length = addresses.length;
-    // accommodating the 2300 gas stipend
-    // adding 1x for each item in array to accomodate rounding errors
-    uint256 gasCost = (23300 * length) + length;
     uint256 balance = address(this).balance;
-    require(balance - gasCost > 10000, "ROYALTIES: Not enough ETH");
-    balance = balance - gasCost;
+    require(balance > 10000, "ROYALTIES: Not enough ETH");
```

Comment:
Should we still keep this check? or at least add a comment why we're limiting payout to be at least 10_000 wei?

Reply:
We should keep this since if we allow for lower than 10000 WEI, we have a chance that some wallets in payouts won't even get anything due to rounding errors. Plus don't see a point triggering royalties for dust amounts.

Reply:
sounds good

Reply:
I removed